backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandra.java

```diff
@@ -177,6 +178,7 @@ public void close() throws IOException {
                         .build()))
             .withTmpFs(ImmutableMap.of("/var/lib/cassandra", "rw,noexec,nosuid,size=200m"))
             .withExposedPorts(CASSANDRA_PORT)
+            .withCreateContainerCmdModifier(createContainerCmd -> createContainerCmd.withName(containerName))
```

Comment:
Likely why you get the error message in your build, as you don't put a random uuid following the name for cassandra (while you seem to do it on others):

```
Status 409: {"message":"Conflict. The container name \"/cassandra_3_11_10-e9b18f2d-9f3f-4eee-b090-2f6058d8654c\" is already in use by container \"03dcdd47a7c1e63c57cea23db20eb54df386249ed19dc2e60b06293b07afd2d1\". You have to remove (or rename) that container to be able to reuse that name."}
```